crates/intern/src/lib.rs

```diff
@@ -94,20 +96,17 @@ impl<T: Internable + ?Sized> Drop for Interned<T> {
 impl<T: Internable + ?Sized> Interned<T> {
     #[cold]
     fn drop_slow(&mut self) {
-        let storage = T::storage().get();
-        let shard_idx = storage.determine_map(&self.arc);
-        let shard = &storage.shards()[shard_idx];
-        let mut shard = shard.write();
-
-        // FIXME: avoid double lookup
-        let (arc, _) = shard.get_key_value(&self.arc).expect("interned value removed prematurely");
```

Comment:
I'm unsure why this was done before. This lookup should yield the input arc, so it's effectively pointless to do, hence why I removed it. (the lock is already taken on the shard as well) I assume this mightve been a refactoring leftover